corehq/apps/smsbillables/tasks.py

```diff
@@ -0,0 +1,40 @@
+from __future__ import absolute_import
+from __future__ import unicode_literals
+
+from datetime import date
+
+from celery.schedules import crontab
+from celery.task import periodic_task
+
+from django.conf import settings
+
+from corehq.apps.smsbillables.models import (
+    SmsBillable,
+    SmsGatewayFeeCriteria,
+)
+from corehq.util.log import send_HTML_email
+from dimagi.utils.dates import add_months_to_date
+
+
+@periodic_task(run_every=crontab(day_of_month='1', hour=5, minute=0), queue='background_queue', acks_late=True)
+def send_gateway_fee_report_out():
+    backend_api_ids = SmsGatewayFeeCriteria.objects.values('backend_api_id').distinct()
```

Comment:
If you change this to
```SmsGatewayFeeCriteria.objects.values_list('backend_api_id', flat=True).distinct()```
then you shouldn't need the `map(lambda ...)` business below

Reply:
Updated - thanks for the tip.